.github/workflows/e2e.yaml

```
@@ -128,6 +128,21 @@ jobs:
           cd internal/testbed/integration/combinedload
           go test -v --tags=e2e
 
+  helm-install-test:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    runs-on: ubuntu-24.04
+    needs: docker-build
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Setup cluster
+        uses: ./.github/actions/create-cluster
+
+      - name: Run helm installation
+        run: |
+          REPOSITORY="dynatrace-otel-collector" TAG="e2e-test" ./internal/testbed/scripts/install_dt_collector_helm.sh || exit 1;
```

you could put the 2 env vars into an `env:` block here
```suggestion
          REPOSITORY="dynatrace-otel-collector" TAG="e2e-test" ./internal/testbed/scripts/install_dt_collector_helm.sh || exit 1;
```